starjudge2/main.py

```diff
@@ -74,38 +70,122 @@ def cine():
+            break
+        
+        elif choice.upper() == "D":
+            conclusion += 0 
+            print("no hiciste ni nada en especial *aplausos \n(FELICIDADES No hay chiste)")
+            break
+        
+        else:
+            print("invalid input")
+    
+            continue
+    
+    print(textC2)
+    input("\npress enter continue\n")
+    #Opción de mejor a peor C2: A, C, B 
+    choice = input("""
+¿Como vas a actuar durante este evento?
+
+A- Divertido
+B- Coqueto
+C- Edgy
+D- ...
+          """)
+    while True:     
+        if choice.upper() == "A":
+            conclusion += A 
+            print("Decidiste no tomarte la situación tan a pecho e hiciste que ambos se rieran de la situación")
+            break     
+        
+        elif choice.upper() == "B":
+            conclusion += C 
+            print("Pusiste una voz seductora y dijiste una frase picardía; \nUn comentario así te hizo ver cómo un payaso, pero la risas no faltaron...")
+            break
             
+        elif choice.upper() == "C":
+            conclusion += B 
+            print("Sin duda alguna, actuar cómo si fueras el cabellero de la venganza te hace ver cool;\n ojalá se pueda decir lo mismo al interactuar directamente contigo...")
+            break
+        
+        elif choice.upper() == "D":
+            conclusion += 0 
+            print("no hiciste ni nada en especial *aplausos \n(FELICIDADES No hay chiste)")
+            break
+        
         else:
             print("invalid input")
+    
             continue
+
+    print(textC3)
+    input("\npress enter continue\n")
+#Opción de mejor a peor C3: B, A, C
+    choice = input("""
+¿Como vas a actuar durante este evento?
+
+A- Divertido
+B- Coqueto
+C- Edgy
+D- ...
+          """)
+    while True:     
+        if choice.upper() == "A":
+            conclusion += B 
+            print("Decidiste no tomarte la situación tan a pecho e hiciste que ambos se rieran de la situación")
+            break     
+        
+        elif choice.upper() == "B":
+            conclusion += A 
+            print("Pusiste una voz seductora y dijiste una frase picardía; \nUn comentario así te hizo ver cómo un payaso, pero la risas no faltaron...")
+            break
+            
+        elif choice.upper() == "C":
+            conclusion += C 
+            print("Sin duda alguna, actuar cómo si fueras el cabellero de la venganza te hace ver cool;\n ojalá se pueda decir lo mismo al interactuar directamente contigo...")
+            break
+        
+        elif choice.upper() == "D":
+            conclusion += 0 
+            print("no hiciste ni nada en especial *aplausos \n(FELICIDADES No hay chiste)")
+            break
         
+        else:
+            print("invalid input")
+    
+            continue     
         
-        return conclusion
+    return conclusion
         
         
-def end(NAME, FRIEND, result):
+def Cine_endings(NAME, FRIEND, result):

```

Comment:
Should be lowercase:

```suggestion
def cine_endings(NAME, FRIEND, result):
```

Reply:
oh, of course, let's make it look mor pythonic